client/stylesheets/contexts/_landing.scss

```diff
@@ -8,8 +8,9 @@
 
 .landing-hero {
   @include grid-container;
-  background-color: $light-grey;
+  background-color: $white;
```

Comment:
You might want to use `background` instead of `background-color` for consistency